.documentation.json

```diff
@@ -10,16 +10,29 @@
     },
     "clearFix",
     "ellipsis",
-    "hiDPI",
     "hideText",
+    "hiDPI",
+    "selection",
+    "placeholder",
+    "retinaImage",
```

Comment:
Those were done in #59 🙈  Would you mind merging that first, rebasing on top of master and then adding the shorthands? Thanks and sorry for the troubles!